backend/app/routes/Q&A/answers/approve.js

```diff
@@ -0,0 +1,18 @@
+const answer = require('../../../models/answers');
+
+module.exports = (req, res, next) => {
+  try {
+    let answerId = req.body.answerId; //getting answer id from body
+    answer.findOne({ _id: answerId }).then((answer) => {
+      if (!answer) {
+        return res.status(404).json({ error: 'answer id not found' });
+      }
+      answer.isApproved = !answer.isApproved;
```

Comment:
Please take this value from `req.body.isApproved`

Reply:
Are you suggesting this.

answer.isApproved = req.body.isApproved;
??

Reply:
yes, as well will take it's value from frontend